tools/generator/tests/test_AnDOGenerator.py

```diff
@@ -79,17 +80,16 @@ def test_complex_parameters(self):
         self.assertEqual(self.guid, ses.guid)
         self.assertEqual(self.sesID, str(ses.sesID))
 
-    # TODO: This check is still failing, review engine/check_Path for consistency
+
     def test_paths(self):
-        ses = AnDOSession(self.expName, self.guid, self.sesID)
+        ses = AnDOSession(self.expName, self.guid, self.sesID, self.ephys)
```

Comment:
This additional argument is not implemented as far as I can see and it also does not really make sense, since the 'ephy' part is fixed for now in the new BEP....

Reply:
Reverted